### PR TITLE
Show community collaborators

### PR DESCRIPTION
**Pull request by nilllzz**

### CODE REVIEW

File: src/app/views/communities/view/overview/overview.vue [hunk 1]
```diff
@@ -22,6 +22,47 @@
...
+						<h5 class="section-header">
+							<translate>Moderators</translate>
+						</h5>
+					</div>
+
+					<div v-for="user of moderators" :key="user.id" class="-mod-list-entry">
+						<app-user-card-hover :user="user">
+							<router-link :to="user.url">
+								<span>
+									{{ user.display_name }}
```
Comment (YLivay) [inline src/app/views/communities/view/overview/overview.vue]: Use the username instead to make it easier for users to address and mention the moderators if needed.

File: src/app/views/communities/view/overview/overview.vue [hunk 2]
```diff
@@ -22,6 +22,47 @@
...
+											class="img-responsive -mod-avatar-img"
+											alt=""
+										/>
+										<app-jolticon class="-mod-verified" v-if="user.is_verified" icon="verified" />
+									</span>
+								</span>
+							</router-link>
+						</app-user-card-hover>
+						<span v-if="owner && user.id === owner.id" class="text-muted small">
+							<translate>owner</translate>
```
Comment (YLivay) [inline src/app/views/communities/view/overview/overview.vue]: Make this a badge.

File: src/app/views/communities/view/overview/overview.vue [hunk 3]
```diff
@@ -22,6 +22,47 @@
 					<br />
 					<app-game-thumbnail :game="community.game" class="-community-game" />
 				</template>
+
+				<div v-if="shouldShowCollabSection" class="-mod-list">
+					<div class="clearfix">
+						<div class="pull-right" v-if="shouldShowLoadMoreCollaborators">
+							<app-button
```
[the remark this answers is not in the document]
Comment (nilllzz) [reply src/app/views/communities/view/overview/overview.vue]: These buttons never are. I already brought this up to David and he said it's fine. Check your user profile, they all look like that.

[the remark this answers is not in the document]
Comment (YLivay) [reply src/app/views/communities/view/overview/overview.vue]: cant unsee

**Pull request merged**